src/Request/PayoutRequest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Devscast\Flexpay\Request;
+
+use Devscast\Flexpay\Credential;
+use Devscast\Flexpay\Data\Currency;
+use Devscast\Flexpay\Data\TransactionType;
+use Webmozart\Assert\Assert;
+
+final class PayoutRequest extends Request
+{
+    public function __construct(
+        float $amount,
+        string $reference,
+        Currency $currency,
+        string $callbackUrl,
+        public string $phone,
+        public Credential $credentials,
+        public string $telephone,
+        public TransactionType $type,
+    ) {
+
+        Assert::length($this->telephone, 12, 'The phone number should be 12 characters long, eg: 243123456789');
+
+        parent::__construct($amount, $reference, $currency, $callbackUrl);
+    }
+
+    public function getPayload(): array
+    {
+        return [
+            'Authorization' => $this->credentials->token,
```

Comment:
authorization != Authorization

`'authorization' => sprintf('Bearer %s', $this->credentials->token)`